google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/mapping/embeddable/EmbeddableImplicitOverrideTest.java

```diff
@@ -9,7 +9,7 @@
 import org.hibernate.annotations.NaturalId;
 import org.hibernate.boot.MetadataBuilder;
 import org.hibernate.boot.model.naming.ImplicitNamingStrategyComponentPathImpl;
-import org.hibernate.testing.junit4.BaseNonConfigCoreFunctionalTestCase;
+import org.hibernate.jpa.test.BaseNonConfigCoreFunctionalTestCase;
```

Comment:
What's the difference in this one vs the one that was there before? Same name.

Reply:
This is a parent test class in the Hibernate libs; we copied it over to our repo under that new package name "org.hibernate.jpa.test" and modified the `HBM2DDL_AUTO` setting in it from 'create-drop' to update. This is what allows us to do the thing where we don't recreate tables.